app/assets/stylesheets/application.css.scss

```diff
@@ -159,3 +159,9 @@ main {
   max-width: 150%;
   margin-left: 0px;
 }
+
+.avatar-show{
+  background-color: black;
+  color: white;
+  box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.7)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space